tests/src/test/scala/system/basic/WskCliActionTests.scala

```diff
@@ -20,9 +20,33 @@ package system.basic
 import org.junit.runner.RunWith
 import org.scalatest.junit.JUnitRunner
 
+import common.TestCLIUtils
+import common.TestUtils.NOT_ALLOWED
 import common.Wsk
 
 @RunWith(classOf[JUnitRunner])
 class WskCliActionTests extends WskActionTests {
   override val wsk = new Wsk
+
+  it should "create an action with an empty file" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
+    val name = "empty"
+    assetHelper.withCleaner(wsk.action, name) { (action, _) =>
+      action.create(name, Some(TestCLIUtils.getTestActionFilename("empty.js")))
+    }
+  }
+
+  it should "not be able to use --kind and --docker at the same time when running action create" in {
+    val file = TestCLIUtils.getTestActionFilename(s"echo.js")
+    val out = wsk.action.create(name = "kindAndDockerAction", artifact = Some(file), expectedExitCode = NOT_ALLOWED,
+        kind = Some("nodejs:6"), docker = Some("mydockerimagename"))
+    out.stderr should include("Cannot specify both --kind and --docker at the same time")
+  }
+
+  it should "not be able to use --kind and --docker at the same time when running action update" in {
```

Comment:
Looks like this test can be folded into the above test allowing variable reuse.

Reply:
Generally I've been using `Seq` to test permutations like below:
```
Seq(false, true).foreach { updateValue =>
  val out = wsk.action.create(name = "kindAndDockerAction", artifact = Some(file), expectedExitCode = NOT_ALLOWED, kind = Some("nodejs:6"), docker = Some("mydockerimagename"), update = updateValue)
  out.stderr should include("Cannot specify both --kind and --docker at the same time")
}
```

Reply:
done